### PR TITLE
Port `Contains` method

### PR DESCRIPTION
**Pull request by saswatamcode**
This ports the `Contains` method from Thanos `testutil` package.

### CODE REVIEW

File: testutil/testutil.go [hunk 1]
```diff
@@ -92,6 +92,52 @@ func FaultOrPanicToErr(f func()) (err error) {
 	return err
 }
 
+// Contains fails the test if needle is not contained within haystack, if haystack or needle is
+// an empty slice, or if needle is longer than haystack.
+func Contains(tb testing.TB, haystack, needle []string) {
```
Comment (bwplotka) [inline testutil/testutil.go:97]: Shouldn't this be.... generic? ;p

Comment (bwplotka) [reply testutil/testutil.go:97]: Otherwise, let's use `ContainsStringSlice` function name?

**Review by bwplotka — APPROVED** [review]
Thanks!

Some comment, WDYT?

File: testutil/testutil.go (hunk 1 above)
Comment (saswatamcode) [reply testutil/testutil.go]: You're right! Something generic would suit here. Let me try to tackle that in next PR. Changing name for now! 🙂

**Review by bwplotka — APPROVED** [review]
Thanks!

**Pull request merged**